### DISCUSSION TITLE
Thanks for this extension!

### DISCUSSION THREAD

**Discussion by ghost** [zero01101/openOutpaint#197] [General]
I want to say thank you for this extension, because it has enabled me to do artworks like this:

![2023-01-30 19 04_39 openOutpaint image](https://user-images.githubusercontent.com/123234324/217157236-2c031fb3-8bd5-4c06-b723-76effbc40a68.png)

![2023-02-06 04 45_39 openOutpaint image](https://user-images.githubusercontent.com/123234324/217157257-ec44016a-7514-4ced-a75a-d91d10b03687.png)

![2023-01-30 05 42_32 openOutpaint image](https://user-images.githubusercontent.com/123234324/217157330-4e1a63c5-7b44-40f4-80d3-49de379a4ef7.png)

![2023-01-30 18 05_54 openOutpaint image](https://user-images.githubusercontent.com/123234324/217157364-8f6beeb2-d0e9-4069-b98d-5636d201aa65.png)

At first, when I was using, I was sure it was some kind of black magic. But, to be honest, it is the future. It is INCREDIBLE what you can do with it.

**Comment by zero01101** [comment]
aw :3 thank you kindly, friendly spooky ghost! <3 if you're still haunting the thread, your landscape pictures are beautiful and i hope you take it as the compliment i mean when i say my mom would _love_ them

**Comment by joshyy1312** [comment]
Hi,

I want to add to this, many, many thanks for this extension, it turned stable diffusion from great to unbelievable for me.

My whole workflow revolves around it (generate base image then back and forth between your extension and Photoshop).

Thanks again :)

![kagawa-shin-1500](https://user-images.githubusercontent.com/59409579/219963362-210deff1-2298-439d-b3f2-ea3514c07b86.png)
![Lleng-Dan-1500](https://user-images.githubusercontent.com/59409579/219963419-9ddf3256-18ec-4541-8134-77115889f549.png)
![Koloss-1500](https://user-images.githubusercontent.com/59409579/219963437-f53b0310-793f-49d8-9cd8-f6934d286517.png)
![NPC-Contact-Amelio-Pancorbo-1500](https://user-images.githubusercontent.com/59409579/219963784-3f6d926c-53cc-4060-b9ae-4b40726d8e31.png)

**Reply by zero01101** [reply]
good lord, those are insane; the middle 3 feel like beltalowda _enforcers_ if you're familiar with the expanse :D hell, all these could fit in an expanse, uh, expansion

**Comment by joshyy1312** [comment]
Thanks :)
I love the expanse , of course.
They are all for a SWN "pen and paper" roleplaying campaign I GM with some friends.
